frontend/web/components/ProjectUsage.tsx

```diff
@@ -0,0 +1,142 @@
+import { FC, useEffect, useState } from 'react'
+import Project from 'common/project'
+import InfoMessage from './InfoMessage'
+// import * as _data from 'common/data/base/_data'
+const _data = require('common/data/base/_data')
+
+type ProjectUsageType = {
+  projectId: string
+}
+
+type getProjectResponseType = {
+  max_features_allowed: number
+  max_segment_overrides_allowed: number
+  max_segments_allowed: number
+  total_features: 11
+  total_segments: 8
+  [key: string]: any
+}
+
+type usageType = {
+  max_features_allowed: number
+  max_segment_overrides_allowed: number
+  max_segments_allowed: number
+  total_features: number
+  total_segments: number
+  total_segment_overrides: {
+    [key: string]: number
+  }
+}
+
+type getEnvironmentsResType = {
+  results: {
+    api_key: string
+    [key: string]: any
+  }[]
+}
+
+type getEnvironmentResType = {
+  [key: string]: any
+  total_segment_overrides: number
+  id: number
+}
+
+const ProjectUsage: FC<ProjectUsageType> = ({ projectId }) => {
+  const [usage, setUsage] = useState<Partial<usageType>>({})
+
+  const getSegmentOverridesUsage = (apiKeys: string[]) => {
+    apiKeys.forEach((apiKey) => {
```

Comment:
This can be replaced with 

  const { data: environment } = useGetEnvironmentQuery({ id: apiKey })

Reply:
I'm not really sure what to do here. I need to get the SegmentOverrideUsage for each project, so i would need to pass an array of apiKeys. the useGetEnvironmentQuery only accepts one apiKey. Any suggestion?